src/memory_raw.c

```diff
@@ -13,15 +13,15 @@ See the file LICENSE for details.
 #include "kernelcore.h"
 #include "pagetable.h"
 
-static uint32_t pages_free = 0;
-static uint32_t pages_total = 0;
-
 static uint32_t *freemap = 0;
 static uint32_t freemap_bits = 0;
 static uint32_t freemap_bytes = 0;
 static uint32_t freemap_cells = 0;
 static uint32_t freemap_pages = 0;
 
+static uint32_t pages_free = 0;
+static uint32_t pages_total = 0;
+
```

Comment:
Why are we moving these? It shouldn't change anything butit doesn't look like we need this file being changed on this PR.